roles/test_operator/README.md

```diff
@@ -25,6 +25,7 @@ Execute tests via the [test-operator](https://openstack-k8s-operators.github.io/
 * `cifmw_test_operator_privileged`: (Boolean) Spawn the test pods with `allowPrivilegedEscalation: true` and default linux capabilities. This is required for certain test-operator functionalities to work properly (e.g.: `extraRPMs`, certain set of tobiko tests). Default value: `true`
 
 ## Tempest specific parameters
+* `cifmw_test_operator_tempest_name`: (String) Value used in the Tempest.Metadata.Name field. The value specifies name of some resources spawned by the test-operator role. Default value: `tempest-tests`
```

Comment:
```suggestion
* `cifmw_test_operator_tempest_name`: (String) Value used in the Tempest.Metadata.Name field. The value specifies the name of some resources spawned by the test-operator role. Default value: `tempest-tests`
```

ditto for each of the cifmw_test_operator_*_name entries.

Reply:
Thank you, done.